pages/talks.js

```diff
@@ -15,42 +15,41 @@ const LoadingCalendar = () =>
             )),
         );
 
-const Talks = () => {
+const NotPlanned = () => {
     const {
-        data: { calendar, notPlanned },
-        isLoading,
+        data: { notPlanned },
         isError,
-    } = useDataApi('/api/calendar', {
-        calendar: [],
-        notPlanned: [],
-    });
-
-    const isLoadingElement = isLoading || isError;
-
-    return (
-        <div>
-            <Layout>
-                <h2>Calendar</h2>
-                <Grid>
-                    {isLoadingElement ? (
-                        <LoadingCalendar />
-                    ) : (
-                        calendar.map((talk, index) => <Talk key={index} talk={talk} />)
-                    )}
-                </Grid>
-                <h2>Not planned</h2>
-                <Grid>
-                    {isLoadingElement ? (
-                        <CardWithLoading />
-                    ) : (
-                        notPlanned.map((talk, index) => (
-                            <Talk key={`notplanned-${index}`} talk={talk} />
-                        ))
-                    )}
-                </Grid>
-            </Layout>
-        </div>
-    );
+    } = useDataApi('/api/notPlanned', { notPlanned: null });
+
+    if (isError) return <CardWithLoading />;
+    if (!notPlanned) return <CardWithLoading />;
```

Comment:
No, we should handle errors :(

Reply:
Done in another PR https://github.com/Euregan/aperoweb/pull/28